src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/Blocks/logo_configuration.html.twig

```diff
@@ -25,102 +25,129 @@
-        </div>
-
-        {{ form_widget(shopLogosForm.header_logo) }}
+      <div class="text-center logo-image-container">
+        <img
+          class="header-logo"
+          src="{{ headerLogoPath }}"
+          alt="{{ 'Header logo'|trans({}, 'Admin.Design.Feature') }}"
+          title="{{ 'Header logo'|trans({}, 'Admin.Design.Feature') }}"
+        >
       </div>
-    </div>
-    <div class="tab-pane" id="invoice-and-mail-configuration" role="tabpanel" aria-labelledby="invoice-and-mail-tab">
-      <div class="form-group">
-        <div class="form-control-label">
-          {{ ps.label_with_help(('Mail logo'|trans({}, 'Admin.Design.Feature')), ('Will appear on email headers. If undefined, the header logo will be used.'|trans({}, 'Admin.Design.Help'))) }}
 
-          <img src="{{ mailLogoPath }}" alt="{{ 'Mail logo'|trans({}, 'Admin.Design.Feature') }}" title="{{ 'Mail logo'|trans({}, 'Admin.Design.Feature') }}">
-        </div>
-
-        {{ form_widget(shopLogosForm.mail_logo) }}
+      {{ ps.form_group_row(shopLogosForm.options.header_logo) }}
+    {% endblock %}
+  {% endembed %}
 
-        {% if headerLogoPath == mailLogoPath %}
-          <p class="form-text  mt-3">{{ 'Warning: if no email logo is available, the main logo will be used instead.'|trans({}, 'Admin.Design.Notification') }}</p>
-        {% endif %}
-      </div>
+  {% embed '@PrestaShop/Admin/Improve/Design/Theme/Blocks/Partials/logo_card.html.twig' %}
+    {% block card_title %}
+      <ul class="nav nav-pills" role="tablist">
+        <li class="nav-item">
+          {{ form_widget(shopLogosForm.options.mail_logo_is_restricted_to_shop) }}
+          <a class="nav-link active show d-inline-block"
+             id="mail-tab"
+             data-toggle="tab"
+             href="#email-configuration"
+             role="tab"
+             aria-controls="home"
+             aria-expanded="true"
+             aria-selected="true"
+          >
+            {{ 'Mail logo'|trans({}, 'Admin.Design.Feature') }}
+          </a>
+        </li>
+        <li class="nav-item">
+          {{ form_widget(shopLogosForm.options.invoice_logo_is_restricted_to_shop) }}
+          <a class="nav-link d-inline-block"
+             id="invoice-tab"
+             data-toggle="tab"
+             href="#invoice-configuration"
+             role="tab"
+             aria-controls="home"
+             aria-expanded="true"
+             aria-selected="true"
+          >
+             {{ 'Invoice logo'|trans({}, 'Admin.Design.Feature') }}
+          </a>
+        </li>
+      </ul>
+    {% endblock %}
+    {% block card_content %}
+      <div class="tab-content">
+        <div class="tab-pane active show" id="email-configuration" role="tabpanel" aria-labelledby="logo-tab">
+          <p class="logo-card-description description-in-tab">
+            {{ 'Will appear on email headers. If undefined, the header logo will be used.'|trans({}, 'Admin.Design.Help') }}
```

Comment:
Hum, it is likely that you inverted the wording between invoice and email. No? Check lines 88 and 105. :-)

Reply:
@LouiseBonnard 
sorry don't understand the problem correctly but I think there is some kind of confusion here:

how it looks now on email tab:

![Screenshot(3)](https://user-images.githubusercontent.com/17051250/54994335-add74600-4fcc-11e9-9a4d-082b17e89401.png)

how it looks now on invoice tab:

![Screenshot(4)](https://user-images.githubusercontent.com/17051250/54994363-bdef2580-4fcc-11e9-9986-4fa692da3615.png)


how it looks on zeplin sketch:

https://zpl.io/2EPr0PY


I think in zeplin there is a mistake since the text is identical to the header logo tab text.

What kind of text should I put there?

Reply:
@tomas862, thanks a lot for the explanation! As checked with @TristanLDD, there is a mistake in the wording of the mockup, it should be:

**Header logo** - Will appear on main page. Recommended size for the default theme: height %height% and width %width%.

**Mail logo** - Will appear on email headers. If undefined, the header logo will be used.

**Invoice logo** - Will appear on invoice headers.

**Favicon** - It is the small icon that appears in browser tabs, next to the title.

Watch the punctuation for the favicon wording, it needs to have a period in the end.

Reply:
done :)